README.md

```diff
@@ -52,6 +53,14 @@ Or install it yourself as:
 $ gem install phobos
 ```
 
+### <a name="upgrade-notes"></a> Upgrade Notes
+
+Version 1.8.2 introduced a new `persistent_connections` setting for
```

Comment:
Nit: Consider removing manual linebreaks as the rest of the readme uses long lines + wrapping to display